common/src/test/java/com/amazonaws/services/schemaregistry/common/configs/GlueSchemaRegistryConfigurationTest.java

```diff
@@ -103,12 +104,28 @@ public void testBuildConfig_noRegionConfigsSupplied_throwsException() {
         assertEquals("Region is not defined in the properties", exception.getMessage());
     }
 
+    /**
```

Comment:
Thanks for this PR. May I suggest you add a negative test case also where DefaultRegionProviderChain throws SDKClientException ?

Reply:
The negative is already there. See [`testBuildConfig_noRegionConfigsSupplied_throwsException`](https://github.com/awslabs/aws-glue-schema-registry/blob/master/common/src/test/java/com/amazonaws/services/schemaregistry/common/configs/GlueSchemaRegistryConfigurationTest.java#L96).

Reply:
@mohitpali Gentle reminder to please have another look.